css/style.css

```diff
@@ -59,18 +67,140 @@ body {
 
 /* Center the container */
 #container {
-  width: 960px;
   margin: auto;
+  padding-right: 20px;
+  padding-left: 40px;
+
 }
 
-header {
+div{
+
+  background-color: #033048;
+  
+  }
+.leftfloat{
+   width:700px;
+   color:white;
+   font-size: 40px;
+   font-weight: bold;
 
 }
+.leftfloat2{
+   width:700px;
+   color:#606161;
+   font-size: 12px;
+   font-weight: bold;
 
-footer {
+}
+.middlefloat{
+  width:20px;
+}
+.rightfloat{
+  width:240px;
+  color:#f9e42e;
+  font-size: 15px
+}
+.rightfloat2{
+  width:240px;
+  color:#606161;
+  font-size: 24px
+}
+.main {
+ float: left;
+ width: 700px;
+}
 
+.sidebar {
+ float: left;
+ width: 240px;
+ border-bottom: none;
+ color: #033048;;
+}
+.topbar{
+  margin:auto;
+  padding-right: 20px;
+  padding-left: 40px;
+}
+section div{
+  background-color: #f0efef;
+  box-sizing:inherit;
+
+}
+section div header{
+  width:700px;
+}
+section div p{
+  width:700px;
+  border-bottom: solid;
+}
+
+header{
+  font-size: 40px;
+  font-weight: semibold; 
```

Comment:
Sorry this was probably a bit confusing. `light`, `semibold` aren't valid CSS values. Instead try using numeric values such as `300` or `600`. 

Valid `font-weight` values can be found here https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#Syntax